modules/Hoot/ui/managePanel/basemaps.js

```diff
@@ -52,18 +53,33 @@ export default class Basemaps extends Tab {
             let basemaps = await Hoot.api.getBasemaps();
 
             this.populateBasemaps( basemaps );
+
         } catch ( e ) {
             window.console.error( 'Unable to retrieve basemaps' );
             throw new Error( e );
         }
     }
 
+    checkBasemapStatus(basemaps) {
```

Comment:
Rather than a forEach to make a new list, what about a filter that returns when the condition you have in the if statement on line 67 is true.

Reply:
and maybe not even a new function, just do that filter on line 78